File: src/FavoDeMel.Api/ClientApp/src/app/comanda/shared/comanda.ts
import { Pedido } from "src/app/pedido/shared/pedido";

export class Comanda {
    id: number;
    nome: string;
    datahoraAbertura: string;
    datahoraFechamento: string;
    totalEmAndamento: number;
    totalFinalizado: number;
    totalPedidos: number;
    pedidos: Pedido
}

File: src/FavoDeMel.Domain/Entities/Comanda.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FavorDeMel.Domain.Entities
{
    public class Comanda
    {
        public Comanda(long id, string nome, DateTime datahoraAbertura)
        {
            Id = id;
            Nome = nome;
            DatahoraAbertura = datahoraAbertura;
        }

        [Key]
        public long Id { get; set; }
        [Required]
        [Column(TypeName = "varchar(100)")]
        public string Nome { get; set; }
        [Required]
        public DateTime DatahoraAbertura { get; set; }
        public DateTime? DatahoraFechamento { get; set; }
        public virtual IList<Pedido> Pedidos { get; set; }
    }
}
